src/coreclr/jit/valuenum.cpp

```diff
@@ -8585,9 +8596,37 @@ void Compiler::fgValueNumberHWIntrinsic(GenTree* tree)
         fgMutateGcHeap(tree DEBUGARG("HWIntrinsic - MemoryStore"));
```

Comment:
@AndyAyersMS   All HW_INTRINSIC Memory Stores mutate the global heap and there are no SIMD store instructions